### PR TITLE
Drop proposal next

### PR DESCRIPTION
**Pull request by JirkaChadima**
This is waiting for https://github.com/windingtree/wt-read-api/pull/126 and https://github.com/windingtree/wt-contracts/pull/198 to be merged

### CODE REVIEW

File: development-handbooks/development.md [hunk 1]
```diff
@@ -42,9 +42,9 @@ a release.
 - `release/*` branches are used when finalizing a release that will go into
 master. All bugfixes related to release branch shall be merged into develop as
 well.
-- `feature/*` branches hold new features, when merged to develop, there should
+- `feature/*` or `feat/*` branches hold new features, when merged to develop, there should
```
Comment (czervenka) [inline development-handbooks/development.md]: this is not related to the drop of proposal/next, right? anyway, shouldn't we be consistent and user `rel/*` instead of `release/*`?

Comment (JirkaChadima) [reply development-handbooks/development.md]: True, this is unrelated. I've just added the `feat/` because we started to use it, because it's consistent with [commitizen](https://raw.githubusercontent.com/commitizen/cz-cli/master/meta/screenshots/add-commit.png).

And we're not (yet) actually using `release` branches. `rel` by itself seems a little ambigous to me, though.
